gRPC/src/main/kotlin/rys/ajaxpetproject/grpc/ChatCreateService.kt

```diff
@@ -0,0 +1,57 @@
+package rys.ajaxpetproject.grpc
+
+import net.devh.boot.grpc.server.service.GrpcService
+import org.slf4j.LoggerFactory
+import reactor.core.publisher.Mono
+import reactor.kotlin.core.publisher.toMono
+import rys.ajaxpetproject.exceptions.BadRequestException
+import rys.ajaxpetproject.exceptions.ChatNotFoundException
+import rys.ajaxpetproject.model.MongoChat
+import rys.ajaxpetproject.request.chat.create.proto.ChatCreateRequest
+import rys.ajaxpetproject.request.chat.create.proto.ChatCreateResponse
+import rys.ajaxpetproject.request.chat.create.proto.ReactorChatCreateServiceGrpc
+import rys.ajaxpetproject.service.ChatService
+import rys.ajaxpetproject.utils.toModel
+import rys.ajaxpetproject.utils.toProto
+
+@GrpcService
+class ChatCreateService(private val chatService: ChatService) :
+    ReactorChatCreateServiceGrpc.ChatCreateServiceImplBase() {
+
+    override fun createChat(request: ChatCreateRequest?): Mono<ChatCreateResponse> {
```

Comment:
```suggestion
    override fun createChat(request: ChatCreateRequest): Mono<ChatCreateResponse> {
```
just use a non-nullable type and you won't need !! later

Reply:
Automatically created. 
Good catch, thanks !